tests/trace/test_op_decorator_behaviour.py

```diff
@@ -141,7 +141,7 @@ def test_sync_method_call(client, weave_obj, py_obj):
         weave_obj_method2 = weave_obj_method_ref.get()
 
     with pytest.raises(errors.OpCallError):
-        res2, call2 = py_obj.amethod.call(1)
+        res2, call2 = py_obj.method.call(1)
```

Comment:
this was a small but in the test suite actually. This should have been testing `method` not `amethod`